### ISSUE TITLE
Test with redis-rb 4.0.0 RC1

### ISSUE THREAD

**Issue by djanowski** [resque/resque-scheduler#590]
I just released redis-rb 4.0.0 RC1.

Can you please test against it and report back?

Thank you!

**Comment by carsonreinke** [comment]
I quickly ran the tests and they pass against resque-scheduler v4.3.0 for resque v1.26.0 and redis-namespace v1.3.0.

I can try some other things too, to see if any issues.

**Comment by carsonreinke** [comment]
Actually ran the tests and seems to of worked: `252 tests, 529 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications`

Using resque-scheduler v4.3.0, resque v1.26.0, and redis-namespace 1.5.2

**Comment by carsonreinke** [comment]
Nevermind, totally failed at this.

There is a couple of failures, let me send them over.

**Comment by carsonreinke** [comment]
Looking into it deeper, most issues are coming directly from resque and resque-namespace:

It appears their two major issues, Redis is sending both these directly to the Redis server:
1. `redis.client.id`
2. `redis.client.reconnect`

**Comment by djanowski** [comment]
Yes, if you absolutely need that, for now the way to do it is by calling `redis._client`.

FYI redis-rb 4.0 is out.

**Comment by ygnessin** [comment]
Is there any update on this?

I am trying to upgrade `redis` gem to 4.x, but to do so I need to upgrade `resque` gem to `2.0`. However I am blocked on that because this gem declares the dependency `resque (~> 1.26)`. 

Perhaps I should open a separate issue?

**Comment by jeremy** [comment]
Resque 2.0 is supported in #661 .

To update the Redis gem to 4.x, you shouldn't need to upgrade Resque, as far as I can tell. What's the blocking dependency?

**Comment by ygnessin** [comment]
@jeremy the blocking change is the following line of code on the 1.27.4 tag of the resque gem:

https://github.com/resque/resque/blob/v1.27.4/lib/resque.rb#L116 

`Redis.connect` was removed in 4.0, so the result is that my rails app crashes.
